File: CS/Form1.cs
namespace DXApplication99 {
    public partial class Form1 : DevExpress.XtraEditors.XtraForm {
        public Form1() {
            InitializeComponent();
            CustomViewInfoRegistrator viewInfoRegistrator = new CustomViewInfoRegistrator();
            navBarControl1.View = viewInfoRegistrator;
        }
    }
}

File: VB/Form1.vb
Namespace DXApplication99
    Partial Public Class Form1
        Inherits DevExpress.XtraEditors.XtraForm

        Public Sub New()
            InitializeComponent()
            Dim viewInfoRegistrator As New CustomViewInfoRegistrator()
            navBarControl1.View = viewInfoRegistrator
        End Sub
    End Class
End Namespace
